ci-operator/step-registry/cucushift/installer/check/vsphere/customized-resource/cucushift-installer-check-vsphere-customized-resource-commands.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+source "${SHARED_DIR}/govc.sh"
+function get_node_names() {
+   local node_type=$1
+   if [ -n "${node_type}" ]; then
+      mapfile -t  node_lists < <(oc get node --no-headers | grep ${node_type} | awk '{print $1}')
+   else
+      mapfile -t  node_lists < <(oc get node --no-headers | awk '{print $1}')
+   fi
+  
+   echo ${node_lists[*]}
+}
+
+function check_master_nodes_number() {
+
+#get node name
+master_lists=$(get_node_names "master")
+IFS=" "
+read -r -a master_lists <<< "${master_lists}"
+#get etcd pods names
```

Comment:
The same as before, we do not need to run any testing against etcd part, in this step, we just validate replicas and customized setting.